mappings/net/minecraft/util/ArraySet.mapping

```diff
@@ -0,0 +1,26 @@
+CLASS net/minecraft/class_4706 net/minecraft/util/ArraySet
+	CLASS class_4707 Iterator
+		FIELD field_21566 start I
+		FIELD field_21567 index I
+	FIELD field_21562 sortingFunction Ljava/util/Comparator;
+	FIELD field_21563 entries [Ljava/lang/Object;
+	FIELD field_21564 capacity I
+	METHOD <init> (ILjava/util/Comparator;)V
+		ARG 0 initialCapacity
+		ARG 1 sortingFunction
+	METHOD method_23859 create (I)Lnet/minecraft/class_4706;
+		ARG 0 capacity
+	METHOD method_23862 addIfNotPresent (Ljava/lang/Object;)Ljava/lang/Object;
+	METHOD method_23863 spliceAdd (Ljava/lang/Object;I)V
+		ARG 0 element
+	METHOD method_23865 peek ()Ljava/lang/Object;
+	METHOD method_23866 toInsertionPoint (I)I
+		ARG 0 wouldBeIndex
+	METHOD method_23868 resizeArray (I)V
+		ARG 0 newCapacity
+	METHOD method_23869 getIndexOf (Ljava/lang/Object;)I
```

Comment:
`index`

Reply:
?

Reply:
`wouldBeIndex` is the return value of `Arrays.binarySearch` where the search found nothing.

The documentation notes that value as `(-(insertion point) - 1)`

```
   private int getIndexOf(T element) {
      return Arrays.binarySearch(this.entries, 0, this.capacity, element, this.sortingFunction);
   }

   private static int toInsertionPoint(int wouldBeIndex) {
      return -wouldBeIndex - 1;
   }

   @Override
   public boolean add(T element) {
      int i = this.getIndexOf(element);
      if (i >= 0) {
         return false;
      }
      this.spliceAdd(element, ArraySet.toInsertionPoint(i));
      return true;
   }
```

Reply:
`java.util.List` uses `indexOf` and not `getIndexOf` for that operation.

Reply:
Yes. I named it as such to avoid conflicting with the inherited method.

Reply:
There should be no inherited method, Sets don't have it.

Reply:
this is not exposed, so index can be used as the cleanest name.

Reply:
`index` is a weird name imo. I would rather have consistency and use the same name as the equivalent List method.